airgun/entities/contentview.py

```diff
@@ -115,6 +140,65 @@ def promote(self, entity_name, version_name, lce_name):
         view.flash.dismiss()
         return view.versions.table.row(version=version_name).read()
 
+    def read_version(self, entity_name, version_name):
+        """Read content view version values"""
+        view = self.navigate_to(
+            self, 'Version',
+            entity_name=entity_name,
+            version_name=version_name,
+        )
+        return view.read()
+
+    def search_version(self, entity_name, query):
+        """Search for content view version"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.versions.search(query)
+
+    def search_version_package(
+            self, entity_name, version_name, query, repo=None):
+        """Search for a package inside content view version
+
+        :param str optional repo: repository name to filter by
+        """
+        view = self.navigate_to(
+            self, 'Version',
+            entity_name=entity_name,
+            version_name=version_name,
+        )
+        return view.rpm_packages.search(query, repo=repo)
+
+    def remove_version(self, entity_name, version_name, completely=True,
+                       lces=None):
+        """Remove content view version.
+
+        :param bool completely: complete content view version removal if True
```

Comment:
what is the difference? Just not sure that I understand what 'completely' word means

Reply:
Should i rephrase? Non-complete content view version removal just 'marks' cvv as deleted and only deletes it from lces, when complete means complete removal from app.

Reply:
You can just put these words into docstring if possible